src/battle_ai_util.c

```diff
@@ -369,21 +369,21 @@ s32 AI_CalcDamageSaveBattlers(u32 move, u32 battlerAtk, u32 battlerDef, u8 *type
 
 static inline s32 LowestRollDmg(s32 dmg)
 {
-    dmg *= MIN_ROLL_PERCENTAGE;
+    dmg *= DMG_ROLL_PERCENT_LO;
     dmg /= 100;
     return dmg;
 }
 
 static inline s32 HighestRollDmg(s32 dmg)
 {
-    dmg *= MAX_ROLL_PERCENTAGE;
+    dmg *= DMG_ROLL_PERCENT_HI;
     dmg /= 100;
     return dmg;
 }
 
 static inline s32 AverageRollDmg(s32 dmg)
 {
-    dmg = ((HighestRollDmg(dmg) + LowestRollDmg(dmg)) * 100) / 2;
+    dmg = (dmg * (DMG_ROLL_PERCENT_LO + DMG_ROLL_PERCENT_HI)) / 2;
```

Comment:
Could you change the equation to always result in the 9th roll of the damage? I brought it up before and ultimately thought it didn't matter much but I think it is crucial for the debug menu to have the correct value shown.

Reply:
The current calculation calcs an average but the average can end up between roll 8 and 9. This doesn't matter for the actual damage calculations but it will affect the values in the debug menu.

Reply:
I still do not follow. What is a "roll 8" and "roll 9?"

Reply:
![image](https://github.com/rh-hideout/pokeemerald-expansion/assets/93446519/7174beb4-4302-48be-9aa6-51b685feb09b)